app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js

```diff
@@ -0,0 +1,151 @@
+angular.module('topologyApp', ['kubernetesUI','ui.bootstrap'])
```

Comment:
the topology changed a lot in the past month implementation wise (refactoring and some new features), so I'm wondering what's the best way to redo this.